core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -330,7 +346,7 @@ def merge_accumulators(
             sum(acc.accepted_translation_word_count for acc in accumulators),
             sum(acc.rejected_translations_count for acc in accumulators),
             sum(acc.rejected_translation_word_count for acc in accumulators),
-            set().union(*[acc.contribution_dates for acc in accumulators])
+            contribution_dates
```

Comment:
Previously MYPY considering the type of ` set().union(*[acc.contribution_dates for acc in accumulators]` is `set[<nothing>]` means it does not contain any value. Also, the type expected by union is Iterable but mypy is not considering `*[...]` as iterable. That's why i have to do these changes.

Reply:
Previously when we are doing `set().union(*[acc.contribution_dates for acc in accumulators])`. Basically, we are first defining an empty set whose type is `Set[<nothing>]` and when we apply union on this set then here we contradict the types. Because the type we are passing in the union is `List[Set[Date]]` but we are using union on set having the type `Set[<nothing>]`. So, the expected type for union is `Iterable[<nothing>]`. To fix this type of contradiction I first defined an empty set having the type Set[datetime.date]:
`contribution_dates: Set[datetime.date] = set()`